ArcGIS-toolbox/ASF_Tools.pyt

```diff
@@ -786,6 +786,7 @@ class RGBDecomp(object):
         indir = parameters[0].valueAsText
         scale = parameters[1].valueAsText
         pol = parameters[2].valueAsText
+        rb_thresh_db = parameters[3].valueAsText
```

Comment:
Change this line to remove the "AsText" part.
rb_thresh_db = parameters[3].value